tutorials/docker-gcplogs-driver/index.md

```diff
@@ -196,7 +192,7 @@ you can skip this section.*
 From [Google Cloud Shell](/shell/docs/quickstart) or a development machine
 where you have [installed and initialized the Google Cloud SDK](/sdk/docs/),
 use the [gcloud compute intances
-add-metadata](https://cloud.google.com/sdk/gcloud/reference/compute/instances/add-metadata)
+add-metadata](/sdk/gcloud/reference/compute/instances/add-metadata)
```

Comment:
It's best to keep the full URL. The relative URL will work in the context of cloud.google.com, but some folks also read the tutorials here on GitHub.